api/app/Traits/Generator/GeneratesUserDoc.php

```diff
@@ -432,6 +464,33 @@ public function experience(Section $section, AwardExperience|CommunityExperience
 
         $this->addLabelText($section, $this->localize('experiences.additional_details'), $experience->details);
 
+        if ($type === WorkExperience::class) {
+            /** @var WorkExperience $experience */
+            if ($experience->employment_category === EmploymentCategory::GOVERNMENT_OF_CANADA->name || $experience->employment_category === EmploymentCategory::CANADIAN_ARMED_FORCES->name) {
+                if ($experience->workStreams) {
+                    $workStreamsByCommunity = [];
```

Comment:
Might need to assert the relation `$experience->workStreams` isn't an empty array other wise it just plops "work streams" in 

![image](https://github.com/user-attachments/assets/d4b4a688-144b-4f8e-925e-a9607da0035e)